src/devices/src/virtio/net/gvproxy.rs

```diff
@@ -34,8 +33,11 @@ impl Gvproxy {
         }
         bind(fd, &local_addr).map_err(ConnectError::Binding)?;
 
-        sendto(fd, &VFKIT_MAGIC, &peer_addr, MsgFlags::empty())
-            .map_err(ConnectError::SendingMagic)?;
+        // Connect so we dnn't need to use the peer address again. This also
```

Comment:
```suggestion
        // Connect so we don't need to use the peer address again. This also
```

Think you meant to add an 'o' here ;)

Reply:
Trying again :-)